universum/analyzers/clang_format.py

```diff
@@ -0,0 +1,67 @@
+import argparse
+import pathlib
+from typing import Callable, List, Optional, Tuple
+
+import yaml
+
+from . import utils, diff_utils
+
+
+def clang_format_argument_parser() -> argparse.ArgumentParser:
+    parser = diff_utils.diff_analyzer_argument_parser("Clang-format analyzer", __file__, "clang-format")
+    parser.add_argument("--executable", "-e", dest="executable", default="clang-format",
+                        help="The name of the clang-format executable, default: clang-format")
+    parser.add_argument("--style", dest="style",
+                        help="The 'style' parameter of the clang-format. Can be literal 'file' string or "
+                             "path to real file. See the clang-format documentation for details.")
+    return parser
+
+
+def _add_style_param(cmd: List[str], settings: argparse.Namespace) -> None:
```

Comment:
Should be named as `_add_style_param_if_present()` or similar, as the parameter is added only if some condition is true.